src/libs/actions/Report.js

```diff
@@ -880,7 +880,6 @@ function deleteReportComment(reportID, reportAction) {
 
     const parameters = {
         reportID,
-        sequenceNumber,
```

Comment:
@aldo-expensify I'm assuming here that we will be making `sequenceNumber` an optional parameter of `DeleteComment` as part of https://github.com/Expensify/Expensify/issues/245243